models/websocket-user.go

```diff
@@ -0,0 +1,9 @@
+package models
+
+import "github.com/gorilla/websocket"
+
+// WebsocketUser holds information for a WebsocketUser user
+type WebsocketUser struct {
+	ID         string          `json:"id"`
+	Connection *websocket.Conn `json:"connection,omitempty"`
```

Comment:
can you please explain why we are storing the connection string to the database?

Reply:
I read online that it's a good practice for microservice architecture. It is basically used for secure communication and service-to-service authorization/authentication. For example, it will ensure that a user can only connect to a WebSocket if they have a valid token.